nix/local/flake.nix

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I dont think this does anything, can we remove it?

Reply:
> I dont think this does anything

[Here](https://github.com/paisano-nix/core/blob/10270dc46532c947de473ee88c8f5a3346a396fb/grow/default.nix#L92C8-L95) is the implementation of the effectuator overloading or adding extra inputs (which in turn don't need to be added / updated c/o the main flake).

Reply:
Can we remove it and use only top-level inputs? I think it's too much to expect from stakeholders to understand how it works and at the end of the day if anything it adds extra storage usage and extra evaluation time (since its unlikely that dependencies will be the same as top-level flake)